HACKING.md

```diff
@@ -67,10 +77,16 @@ helper classes.
 
 #### `flutter.py`
 
+Tool to verify and mark validity for any Giza-style YAML input.
```

Comment:
More generally, flutter is a tool that loads arbitrary JSON/YAML/TOML data into a class and reports schema violations. It's also used for configuration files and such, not just Giza files!

Reply:
good catch! thanks for the clarification. updated